.eslintrc

```diff
@@ -1,5 +1,9 @@
 {
   "root": true,
+  "env": {
+    "browser": true,
```

Comment:
Why would we set `browser` true for a nodejs/backend project?

Reply:
yes - my mistake :) changed it